Makefile

```diff
@@ -43,6 +47,7 @@ help:
 	@echo "  docserve           run an HTTP server on the docs directory"
 	@echo "  html               make HTML docs"
 	@echo "  htmlview           open the index page built by the html target in your browser"
+	@echo "  htmllive           to rebuild and reload HTML files in your browser"
```

Comment:
```suggestion
	@echo "  htmllive           rebuild and reload HTML files in your browser"
```
?

Reply:
I understand it's different from the docs/Makefile, but it's consistent with the rest of this Makefile?